packages/grafana-ui/src/components/Table/FilterPopup.tsx

```diff
@@ -95,7 +79,7 @@ export const FilterPopup = ({
           <Stack gap={3}>
```

Comment:
```suggestion
          <Stack justifyContent={'space-between'}>
```

I think it looks better 😄 

Before:
<img width="463" alt="Screenshot 2024-09-30 at 14 07 44" src="https://github.com/user-attachments/assets/6a431137-bd3f-4da1-96be-d3d68541c6bb">

After:
<img width="454" alt="Screenshot 2024-09-30 at 14 07 50" src="https://github.com/user-attachments/assets/f955e197-e60a-4d29-bb34-93d48c689837">